docs/modules/mechanics/gamerules.mdx

```diff
@@ -23,28 +19,38 @@ When stopping the daylight cycle, you should either use the `<doDaylightCycle>`
 | `<doFireTick>` | Toggles whether fire spread occurs. | <span className="badge badge--primary">true/false</span> |
 | `<doTileDrops>` | Toggles whether tile entities drop. | <span className="badge badge--primary">true/false</span> |
 | `<doMobLoot>` | Toggles whether mobs drop loot on death. | <span className="badge badge--primary">true/false</span> |
+| `<keepInventory>` | Toggles whether players should keep their inventory content after death.<br />*Some maps use this to prevent item drops on death without adding them to [Item Remove](/docs/modules/gear/repair-remove-keep#item-remove).* | <span className="badge badge--primary">true/false</span> |
```

Comment:
In PGM, keep inventory gamerule won't actually make you keep anything, which should probably be clarified. If you want keep inventory mechanics, you need to use pgm's keep inventory module, the *only* purpose of keep inventory gamerule is preventing items dropping on death